src/app/screens/Send/index.tsx

```diff
@@ -95,11 +95,16 @@ function Send() {
           },
         });
       } else {
-        lightningPayReq.decode(invoice); // throws if invalid.
+        let bolt11, match;
+        // look for bolt11 encoding if it's a BIP21 invoice
+        if ((match = invoice.trim().match(/bitcoin:\S*lightning=([^&]*)/i))) {
```

Comment:
Can you move this to the outer if statement so it reads as "another" method? (like `isPubKey()` above)

Reply:
can't we reuse the `extractInvoiceFrom()` function here? that should be enough and we already use it for the QR code.

Reply:
Ah yes, that makes it easy!